build.gradle.kts

```diff
@@ -1,27 +1,24 @@
 plugins {
     id("org.hypertrace.repository-plugin") version "0.4.0"
     id("org.hypertrace.ci-utils-plugin") version "0.3.0"
+    id("org.hypertrace.docker-java-application-plugin") version "0.9.0" apply false
+    id("org.hypertrace.docker-publish-plugin") version "0.9.0" apply false
```

Comment:
Can we remove the publish plugins for now? Once we have some working version, we will add it. Remove both - docker, and lib.